client/quantum_serverless/core/program.py

```diff
@@ -76,6 +76,29 @@ def from_json(cls, data: Dict[str, Any]):
         return Program(**{k: v for k, v in data.items() if k in field_names})
 
 
+@dataclass
+class Configuration:  # pylint: disable=too-many-instance-attributes
+    """Program Configuration.
+
+    Args:
+        title: program name
+        entrypoint: is a script that will be executed as a job
+            ex: job.py
+        env_vars: env vars
+        dependencies: list of python dependencies to execute a program
+        working_dir: directory where entrypoint file is located (max size 50MB)
+        description: description of a program
+        version: version of a program
```

Comment:
should the args here be workers, cpu, etc.?

Reply:
The configuration is specified in `run` method of `Quantum_serverless` 
```
serverless.run(program, config=Configuration(max_workers=12, min_workers=5)
```

I'll update the doc string of the function